.storybook/Divider.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withInfo } from '@storybook/addon-info'
+import { Divider } from '../src'
+
+const description = '<hr /> with settings for padding, margin, width, and borderColor'
+
+storiesOf('Divider', module)
+  .add('Divider component', withInfo({
+    text: description,
+    inline: true
+  })(() => (
+    <Divider borderColor='lightGray' />
```

Comment:
Just noticed you've got the `borderColor` prop on all these – let's set that as a defaultProp on Divider so there's a smart default. Also would be good to rename that value to `borderGray` since I merged #92 in

Reply:
Updated @jxnblk